packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -52,14 +69,4 @@ const SettingsHeader = ( {
 	);
 };
 
-export default withDispatch( ( dispatch ) => {
-	const { openGeneralSidebar } = dispatch( 'core/edit-post' );
-	return {
-		openDocumentSettings() {
-			openGeneralSidebar( 'edit-post/document' );
-		},
-		openBlockSettings() {
-			openGeneralSidebar( 'edit-post/block' );
-		},
-	};
-} )( SettingsHeader );
```

Comment:
This change is not relevant. Just cleaning up to use hooks instead of HoC.

Reply:
Nice!